### ISSUE TITLE
Games loaded through arcade library only work on first launch

### ISSUE THREAD

**Issue by dfbills** [OpenEmu/OpenEmu#3601]
I've found that ROMs loaded into 2.0.6.1 experimental using the Arcade library appear to work fine on first import and launch.  Once the initial session is complete, launching the games gives a black, unresponsive screen.  The only way out is to terminate the application.

**Comment by clobber** [comment]
[Contributing guidelines](https://github.com/OpenEmu/OpenEmu/blob/master/CONTRIBUTING.md)
- "Please search before opening a new issue."
- "Arcade/MAME is unsupported because it is an experimental core port and has inherent usability and maintainability issues. It is recommended to use the official [standalone](http://bamf2048.github.io/sdl_mame_tut/) version and to read this helpful [MAME ROMs tutorial](http://choccyhobnob.com/articles/demystifying-mame-roms/)."

Duplicate of https://github.com/OpenEmu/OpenEmu/issues/2615#issuecomment-230357101

Duplicate of https://github.com/OpenEmu/UME-Core/issues/7
